fortnite_python/domain.py

```diff
@@ -2,38 +2,48 @@
 
 
 class Platform(Enum):
+    """Defines the platform the account uses"""
```

Comment:
This is to refer to the platform always the same way and to prevent the changes if the api updates.